### PR TITLE
[mongorito] Typing for version 3.0

### PR DESCRIPTION
**Pull request by pinguet62**
Hello,

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

Have a nice day.

### CODE REVIEW

File: types/mongorito/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,297 @@
+// Type definitions for mongorito v3.0.4
+// Project: https://github.com/vadimdemedes/mongorito
+// Definitions by: Pinguet62 <https://github.com/pinguet62>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import {Collection, CommonOptions, Db, IndexOptions, Long, MongoClientOptions, ReadPreference} from 'mongodb';
+
+declare namespace mongorito {
+}
```
Comment (plantain-00) [inline types/mongorito/index.d.ts]: It seems the empty namespace can be removed

File: types/mongorito/mongorito-tests.ts [hunk 2]
```diff
@@ -0,0 +1,553 @@
+/**
+ * - Compilation type-check: using TS compiler "strict" options, and DefinitelyTyped linter (based on comments)
+ * - Dynamic type-check: using `expect(...).toBeInstanceOf(...)` on a simple workflow (require Mongodb instance)
+ */
```
Comment (plantain-00) [inline types/mongorito/mongorito-tests.ts]: It seems they can be removed

File: types/mongorito/tslint.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```
Comment (plantain-00) [inline types/mongorito/tslint.json]: For new packages, this file should only `"extends": "dtslint/dt.json"`

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/mongorito/index.d.ts (hunk 1 above)
Comment (pinguet62) [reply types/mongorito/index.d.ts]: removed

File: types/mongorito/mongorito-tests.ts (hunk 2 above)
Comment (pinguet62) [reply types/mongorito/mongorito-tests.ts]: removed

File: types/mongorito/tslint.json (hunk 3 above)
Comment (pinguet62) [reply types/mongorito/tslint.json]: Updated, and linter issues fixed

**Review by plantain-00 — APPROVED** [review]

File: types/mongorito/index.d.ts [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+    INCREMENT = "@@mongorito/INCREMENT",
+    CREATE_INDEX = "@@mongorito/CREATE_INDEX",
+    DROP_INDEX = "@@mongorito/DROP_INDEX",
+    LIST_INDEXES = "@@mongorito/LIST_INDEXES",
+    QUERY = "@@mongorito/QUERY",
+    CALL = "@@mongorito/CALL"
+}
+
+export interface GetAction {
+    type: string; // ActionTypes.GET
```
Comment (uniqueiniquity) [inline types/mongorito/index.d.ts]: You can actually make this `type: ActionTypes.GET` and enforce that `@@mongorito/GET` is the only allowable value.

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

File: types/mongorito/index.d.ts (hunk 4 above)
Comment (pinguet62) [reply types/mongorito/index.d.ts]: Very nice remark! (I previously searched... but not found this simple solution)
The PR is now _up to date_

**Review by uniqueiniquity — APPROVED** [review]

**Pull request merged**